landscaper/common/generate/blueprint/generate.go

```diff
@@ -42,21 +42,23 @@ const (
 	targetSchema = "com.github.gardener.landscaper.apis.core.v1alpha1.Target.yaml"
 )
 
-// BlueprintImport is an import field in the rendered blueprint
```

Comment:
Is the `_ "embed"` import in this file actually needed/used? I don't see any `//go:embed` directive in this file.

Reply:
Thanks for the find.